app/domain/operations/families/verifications/dmf_determination/build_cv3_family_payload_for_dmf.rb

```diff
@@ -0,0 +1,130 @@
+    module Verifications
+      module DmfDetermination
+        # operation to build cv3 family to publish to fdsh
+        class BuildCv3FamilyPayloadForDmf
+          include Dry::Monads[:result, :do]
+          include ::Operations::Transmittable::TransmittableUtils
+
+          # @return [ Cv3Family ] Job successfully completed
+          def call(family, transmittable_params)
+            @job = transmittable_params[:job]
+            @transmission = transmittable_params[:transmission]
+            @transaction = transmittable_params[:transaction]
+            @family = family
+
+            cv3_family = yield build_cv3_family
+            valid_cv3_family = yield validate_cv3_family(cv3_family)
+
+            yield validate_all_family_members(valid_cv3_family)
+            result = yield confirm_transmittable_payload(cv3_family)
+
+            Success(result)
+          end
+
+          private
+
+          def build_cv3_family
+            cv3_family = Operations::Transformers::FamilyTo::Cv3Family.new.call(@family)
+            return handle_dmf_failure("Unable to transform family into cv3_family: #{cv3_family.failure}", :build_cv3_family) if cv3_family.failure?
+
+            cv3_family
+          end
+
+          def validate_cv3_family(cv3_family)
+            valid_cv3_family = AcaEntities::Operations::CreateFamily.new.call(cv3_family)
+            return handle_dmf_failure("Invalid cv3 family: #{valid_cv3_family.failure}", :validate_cv3_family) if valid_cv3_family.failure?
+
+            valid_cv3_family
+          end
+
+          def validate_all_family_members(aca_family)
+            invalid_persons = aca_family.family_members.map do |aca_member|
+              family_member = @family.family_members.detect { |fm| fm.hbx_id == aca_member.hbx_id }
+
+              # check if member does not have an enrollment
+              unless member_eligible?(aca_member)
+                vh_message = "Family Member with hbx_id #{aca_member.hbx_id} does not have a valid enrollment"
+                update_verification_type_histories(vh_message, [family_member])
+                next vh_message
+              end
+
+              # check if member is valid (valid_ssn, etc.)
+              valid_member = Operations::Fdsh::PayloadEligibility::CheckPersonEligibilityRules.new.call(aca_member.person, :alive_status)
+              next if valid_member.success?
+
+              vh_message = "Family Member with hbx_id #{aca_member.hbx_id} is not valid: #{valid_member.failure}"
+              update_verification_type_histories(vh_message, [family_member])
+              vh_message
+            end.compact
+
+            # invalid_persons.size == family_members.size indicates no family members were valid
+            return Success(aca_family) unless invalid_persons.size == aca_family.family_members.size
+
+            message = "No Family Members valid: #{invalid_persons.join(', ')}"
+            handle_dmf_failure(message, :build_cv3_family)
```

Comment:
This will record the failure in verification history twice.

1.  https://github.com/ideacrew/enroll/pull/3888/files#diff-1abc7c0ba548fe62b2c2a24dae17883e1ccc49d5bc0b4f22c74c2225c1761235R63 
or 
https://github.com/ideacrew/enroll/pull/3888/files#diff-1abc7c0ba548fe62b2c2a24dae17883e1ccc49d5bc0b4f22c74c2225c1761235R54

2. In this call handle_dmf_failure